jbehave-support-core/docs/Web-testing.md

```diff
@@ -271,4 +271,29 @@ Then on [home] page wait until [@title] has text Google
 ```
 For more examples see [WebWaitCondition.story](../src/test/groovy/org/jbehavesupport/test/sample/WebWaitCondition.story)
 
+#### Taking screenshot
+
+To take screenshot 
+```
+Then screenshot is taken
+When screenshot is taken
+```
+
+If you want to take screenshot from your custom web steps inject `ApplicationEventPublisher` and publish `WebScreenshotEvent`
+```
+import static org.jbehavesupport.core.web.WebScreenshotTypes.*;
+
+@Autowired
+ApplicationEventPublisher applicationEventPublisher;
+
+public void myMethod(){
+    ...
+    applicationEventPublisher.publishEvent(new WebScreenshotEvent(this, Type.MANUAL));
+    ...
+}
+```
+
+If you want to see screenshots in report register `ScreenshotReporterExtension`
+ * Type have to be `MANUAL` or correspond with chosen mode, else screenshot won't be taken
```

Comment:
```suggestion
 * Type has to be `MANUAL` or correspond with chosen mode (see property `web.screenshot.reporting.mode` in [Reporting](Reporting.md)), otherwise screenshot won't be taken
```

Reply:
Thanks :)